beginner-guide.md

```diff
@@ -0,0 +1,61 @@
+# A Beginner's Guide to using S3DF 
```

Comment:
this doesn't appear too different in content from the README.md. is the intention to extend this with more details with infographics/flow charts etc.